Automattic/wp-calypso
# Autofix wpcalypso/no-package-relative-imports in client/extensions

### Background

See #44602

### Changes

Automatically apply fix for eslint rule `wpcalypso/no-package-relative-imports` in `./client/extensions`

Fix applied with `./node_modules/.bin/eslint-nibble --ext .js,.jsx,.ts,.tsx,.md.js,.md.javascript,.md.jsx client/extensions`

### Testing instructions

* Check all tests passes (ignore lint errors, there will be a ton).
* Do a smoke test on the live branch
* Verify there are no huge changes in package size in ICFY report.

###
Notes:

- To test the zoninator extension you either go to `/extensions/zoninator` on an Atomic site, or go to Plugin management in a Jetpack site.
- To test the woocommerce extension you either go to `/store` on an Atomic site, or click on 'Shop' in the navigation bar in a site with the `Business` plan
